searchcore/src/vespa/searchcore/fdispatch/search/plain_dataset.cpp

```diff
@@ -196,7 +195,7 @@ FastS_PlainDataSet::FastS_PlainDataSet(FastS_AppContext *appCtx,
                                        FastS_DataSetDesc *desc)
     : FastS_DataSetBase(appCtx, desc),
       _partMap(desc),
-      _stateOfRows(_partMap.getNumRows(), 1.0, desc->GetQueryDistributionMode().getLatencyDecayRate()),
+      _stateOfRows(_partMap.getNumRows(), 0.010, desc->GetQueryDistributionMode().getLatencyDecayRate()),
```

Comment:
need to make sure the initial value is lower than any realistic latency to make sure all groups are being used in the normal case, not only as a fallback when the other groups break down.

Reply:
Considered and done.